### PR TITLE
Add Utils for Dictionaries

### PR DESCRIPTION
**Pull request by mikaell**
This PR adds a functionality: add a file for dictionary utils. This functionality already exists in the code base. This is an effort to gather it in one place for reusability. Future pr:s can refactor and start using this utility file.

**How to test**:
1. how to test it, possibly with real cases/data
Pytests should pass

Usage example:
```
>>> import dict_utils
>>> a = {'asd': 12, 'a': 123, 'k': 33, '1': None, '2': []}
>>> dict_utils.remove_nonetype(a)
{'asd': 12, 'a': 123, 'k': 33, '2': []}
```
**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: tests/utils/test_dict_utils.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    d = {"a": "1", "b": 2, "c": None}
+
+    # THEN calling removeNoneValues(dict) will remove key-value pair
+    # where value=None
+    assert {"a": "1", "b": 2} == remove_nonetype(d)
+
+
+def test_remove_empty_list_no_change():
+    # WHEN a dict *not* containing an empty list, []
+    d = {"a": "1", "b": 2, "c": 3}
```
Comment (northwestwitch) [inline tests/utils/test_dict_utils.py:23]:
```suggestion
    d = {"a": "1", "b": 2, "c": [3]}
```

Comment (northwestwitch) [reply tests/utils/test_dict_utils.py:23]: It's fine really, but perhaps it's a better test if you provide an element with an array?

**Review by northwestwitch — APPROVED** [review]
Looks good!

File: tests/utils/test_dict_utils.py (hunk 1 above)
Comment (mikaell) [reply tests/utils/test_dict_utils.py]: Good idea. Changes are being pushed.

Comment (northwestwitch) [reply tests/utils/test_dict_utils.py]: It's good to merge I think!

**Pull request merged**